### DISCUSSION TITLE
Welcome to Miele Integration Discussions!

### DISCUSSION THREAD

**Discussion by astrandb** [astrandb/miele#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by DScholz** [comment]
Hi, i have a new DA 2698 (hood). The integration works fine, but i have a little problem with the integrated light's, they are able to dim via the key‘s (press the key a bit longer) on the hood or with the app (slider). Is it possible to get the function in HA?

Thanks in advance
[miele-0c9c033c1425c3daea71b6a6525a645a-Dunstabzugshaube-adb59527a69e6b52c508699c5aebfadb(5).json](https://github.com/user-attachments/files/15530790/miele-0c9c033c1425c3daea71b6a6525a645a-Dunstabzugshaube-adb59527a69e6b52c508699c5aebfadb.5.json)

**Reply by astrandb** [reply]
No, the Miele API only allows on/off for lights.
